mappings/net/minecraft/client/render/entity/EntityRendererExplodingWitherSkull.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS none/bvb net/minecraft/client/render/entity/EntityRendererExplodingWitherSkull
-	FIELD a INVINSABLE_SKIN Lnone/kp;
+	FIELD a INVINCIBLE Lnone/kp;
```

Comment:
Oh sorry, I just meant to point out the typo, not that you should get rid of `_SKIN`.